lib/restclient/request.rb

```diff
@@ -536,7 +536,18 @@ def log_request
       return unless RestClient.log
 
       out = []
-      out << "RestClient.#{method} #{url.inspect}"
+      sanitized_url = begin
```

Comment:
@xaviershay Might be nice to extract `sanitized_url` as a method to encourage reuse. But NBD.